addons/armaos/functions/fnc_computer_processActionUse.sqf

```diff
@@ -0,0 +1,13 @@
+/**
+ * Gets triggered by ACE action defined in the config of the computer object. Calls the terminal init fuction.
```

Comment:
Why is this function needed?

Reply:
It's more like a wrapper. I thought there will be perhaps some additional code. But that never happend. So your right. It's unnecessary and I will remove it.

Reply:
deleted in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763